doc/nrf/libraries/others/pcm_mix.rst

```diff
@@ -0,0 +1,32 @@
+.. _lib_pcm_mix:
+
+Pulse Code Modulation audio mixer
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Pulse Code Modulation (PCM) audio mixer lets you mix a PCM stream into another PCM stream.
+This library is useful for developing applications that offer audio features, for example using the nRF5340 Audio DK.
+
+The library offers the following types of mixing support:
+
+* Stereo stream into another stereo stream
+* Combinations of mono to mono
+* Mono to stereo channel left or right
```

Comment:
Is "channel left or right" referring to "stereo" only or to both "mono and stereo"?

Reply:
I don't quite understand. You can mix a mono stream to stereo-L or stereo-R or stereo L+R.
See updated version which I will push before lunch.